### PR TITLE
Rename files, create signup mutation, create a class method in user

### PR DESCRIPTION
**Pull request by matydn05**
I was able to make it all work in the signInResolver. I didn't want to try to write it all in user.js because I thought it would make more sense if it was done the same way I just did the signUpResolver . If user.js is the one who should have those things then I guess I could try to change both of them ?

### CODE REVIEW

File: src/models/user.js [hunk 1]
```diff
@@ -37,6 +37,13 @@ export default (sequelize, DataTypes) => {
   }
 
   // Class methods
+  User.getNonSensibleInformation = async function (user) {
+    return {
+      username: this.username,
```
Comment (abelosorio) [inline src/models/user.js]: In this context, `this` represents the `User` class (not an instance). So, you should replace `this` by `user`.

**Review by abelosorio — COMMENTED** [review]
Looks good. I left a comment above. And yes, you're on the right way. I'd suggest moving the signin/signup logic to the User class and try to refactor it in small steps.

File: src/models/user.js (hunk 1 above)
Comment (matydn05) [reply src/models/user.js]: I wrote it that way the first time and it would not work. I even tried to use the user as an argument but still nothing.

**Review by abelosorio — APPROVED** [review]
:+1: perfect!

**Pull request merged**